src/components/molecules/ProfilePictureInput/ProfilePictureInput.tsx

```diff
@@ -27,6 +33,7 @@ export interface ProfilePictureInputProps {
   pictureUrl: string;
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   register: any;
+  githubHandle?: string;
```

Comment:
If we just added this directly to the user profile then we could read it off `user`

Reply:
At the point, when `ProfilePictureInput` is rendered, the user object is not yet updated and we retrieve this data from LocalStorage. In future, its value used as `partyName` and there is no sense in duping it

Reply:
Fair. Makes sense.